app/assets/javascripts/beak/widgets/event-traffic-control.coffee

```diff
@@ -233,27 +241,63 @@ controlEventTraffic = (controller, performUpdate) ->
     world.changeTopology(wrapX, wrapY)
     return
 
-  mousetrap = Mousetrap(ractive.find('.netlogo-model'))
-  mousetrap.bind(['up', 'down', 'left', 'right']            , (_, name) -> ractive.fire('nudge-widget', name))
-  mousetrap.bind(['ctrl+shift+l', 'command+shift+l']        ,           -> ractive.fire('toggle-interface-lock'))
-  mousetrap.bind(['ctrl+shift+h', 'command+shift+h']        ,           -> ractive.fire('hide-resizer'))
-  mousetrap.bind('del'                                      ,           -> ractive.fire('delete-selected'))
-  mousetrap.bind('escape'                                   ,           -> ractive.fire('deselect-widgets'))
+  setTab = (tabName, options = {}) ->
+    tabCanonicalName = tabName.charAt(0).toUpperCase() + tabName.slice(1).toLowerCase()
 
-  mousetrap.bind('?', onQMark)
+    validTabs = ['Console', 'Code', 'Info']
+    unless tabCanonicalName in validTabs
+      console.error("Invalid tab name: #{tabName}. Valid tabs are: #{validTabs.join(', ')}")
+      return
+
+    showPropertyName = "show#{tabCanonicalName}"
+
+    if options.active is 'toggle'
+      showPropertyValue = not ractive.get(showPropertyName)
+    else if typeof options.active isnt 'undefined'
+      showPropertyValue = options.active
+    else
+      return
 
+    ractive.set(showPropertyName, showPropertyValue)
+
+    if showPropertyValue
+      componentToFocus = {
+        "Code": "codeEditor",
+        "Info": "infoeditor",
+        "Console": "console"
```

Comment:
Is this right?  One is `codeEditor` (capital `E`), while the other is `infoeditor` (lowercase `e`)?

Reply:
This should be made consistent.